### DISCUSSION TITLE
Auto rejection after I fixed mistakes

### DISCUSSION THREAD

**Discussion by Lesnoy40rt77** [community/community#46424] [GitHub Education] [answered]
Hello, I am student from Russia and I was trying to appeal for GitHub Student Pack using an official notice from my school, but was rejected due to untranslated notice and lack of information in my profile. Now, I completed my profile, but every time I still get auto-rejected, even if I try to upload my student ID or translated notice. What should I do to stop getting auto-rejections?

**Comment by jge162** [comment, marked as answer]
@Lesnoy40rt77 You need to just submit a GitHub education support ticket. They can review your case. -> [here](https://support.github.com/request/education?tags=hubberfy_education)
